src/core/reducers/categories.js

```diff
@@ -91,24 +91,24 @@ export function createEmptyCategoryList(): EmptyCategoryListType {
       ...object,
       [appName]: validAddonTypes.reduce((appObject, addonType) => {
         // This is where we define an `EmptyCategoryListType` and not a
-        // `CategoryListType`, because `[addonType]` is initialized with an
+        // `CategoryMapType`, because `[addonType]` is initialized with an
```

Comment:
Nit: I'm not sure we even need this comment anymore, now that they have different names. I think the code might be understandable enough without it, but I'll leave that up to you.

Reply:
that's correct